http_header_format_rationale.md

```diff
@@ -1,170 +0,0 @@
-# Trace context HTTP header format rationale
```

Comment:
Is this entire file related to response headers?  It looks like it talks about traceparent and tracecontext, and I see the response headers section only later in this file: should only that particular section be deleted instead of the entire file?

Reply:
This was a mistake

Reply:
Fixed